### DISCUSSION TITLE
Week 3 Discussion Post - Cooper Sanders

### DISCUSSION THREAD

**Discussion by coopsand** [USU-ITLS-x265-23-summer/class-repo#90] [Week 3 - Discussions]
### Hello Everyone!

I am going to start off right away by saying that my feelings on FreeCodeCamp haven't really changed. I still prefer FreeCodeCamp's teaching method over Exercism's, and appreciate the step-by-step approach they take to teaching.

I really enjoyed the games that we got to play this week. I spent most of my time playing CSS Diner, so that's what I'll talk about the most here. One thing I really loved is that it took you step by step, and each level only got a _little_ harder than the previous. I found it helpful that each level built on the ideas of the last, and didn't change skills abruptly.

One suggestion that I have for CSS Diner is to add a tutorial graphic. I spent way too long looking for the goal of the first level than I should have. (I assumed it was in the right panel. It's at the top of the screen.) Once I figured that out, I also wasn't _exactly_ sure where I needed to type in the code, so I guessed and luckily it worked out. 

I feel that I will be revisiting this game through the semester because I have found it very helpful. I also am hoping to spend a little more time in the other game this upcoming week.

**Comment by derekscholes** [comment]
Yeah, I went through it again and it made slightly more sense then previously. I still think I need to go through it again so that I can remember each of different selectors in CSS
